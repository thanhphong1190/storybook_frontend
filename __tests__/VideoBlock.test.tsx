import {
  VideoBlock,
  VideoBlocksProps
} from '../src/components/blocks/VideoBlock'
import renderer from 'react-test-renderer'

const props:VideoBlocksProps = {
  leftThumbnailURL: '/example/videos-example.png',
  leftVideoTitle: `BECOME A MEMBER`,
  leftVideoURL: 'https://www.youtube.com/embed/y8Ok76X15Xk',
  leftVideoDesc: `orem ipsum dolor sit amet, consectetur<br />adipiscing elit, sed do eiusmod tempor`,
  rightThumbnailURL:'/example/heroImg-example.jpg',
  rightVideoTitle: 'OWN A GYM',
  rightVideoURL: 'https://www.youtube.com/embed/ucESGeAlUk4',
  rightVideoDesc: `Kathy Cummings<br />Plus fitness franchisee`
}

it('renders correctly', () => {
  const tree = renderer.create(<VideoBlock {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
