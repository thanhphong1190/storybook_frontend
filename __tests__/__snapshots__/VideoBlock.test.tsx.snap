// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="VideoBlockstyled__VideoWrapper-hfhlfy-0 gedKOy"
>
  <div
    className="VideoBlockstyled__PlayerWrapper-hfhlfy-1 hMDjlQ"
  >
    <h2
      className="VideoBlockstyled__VideoTitle-hfhlfy-2 eurhR"
    >
      BECOME A MEMBER
    </h2>
    <a
      className="VideoBlockstyled__PlayButtonWrap-hfhlfy-4 jiqxxZ"
      onClick={[Function]}
    >
      <img
        src="test-file-stub"
      />
    </a>
    <p
      className="VideoBlockstyled__VideoDescription-hfhlfy-3 dFiQeE"
      dangerouslySetInnerHTML={
        Object {
          "__html": "orem ipsum dolor sit amet, consectetur<br />adipiscing elit, sed do eiusmod tempor",
        }
      }
    />
  </div>
  <div
    className="VideoBlockstyled__PlayerWrapper-hfhlfy-1 jdqQgN"
  >
    <h2
      className="VideoBlockstyled__VideoTitle-hfhlfy-2 eurhR"
    >
      OWN A GYM
    </h2>
    <a
      className="VideoBlockstyled__PlayButtonWrap-hfhlfy-4 jiqxxZ"
      onClick={[Function]}
    >
      <img
        src="test-file-stub"
      />
    </a>
    <p
      className="VideoBlockstyled__VideoDescription-hfhlfy-3 dFiQeE"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Kathy Cummings<br />Plus fitness franchisee",
        }
      }
    />
  </div>
</div>
`;
